scala/scala.bzl

```diff
@@ -409,13 +409,33 @@ def write_manifest(ctx):
         output=ctx.outputs.manifest,
         content=manifest)
 
+def _path_is_absolute(path):
+    # Returns true for absolute path in Linux/Mac (i.e., '/') or Windows (i.e.,
+    # 'X:\' or 'X:/' where 'X' is a letter), false otherwise.
+    if len(path) >= 1 and path[0] == "/":
+        return True
+    if len(path) >= 3 \
+            and path[0].isalpha() \
+            and path[1] == ":" \
+            and (path[2] == "/" or path[2] == "\\"):
+        return True
+
+    return False
+
 def _write_launcher(ctx, rjars, main_class, jvm_flags, args="", wrapper_preamble=""):
     runfiles_root = "${TEST_SRCDIR}/%s" % ctx.workspace_name
     # RUNPATH is defined here:
     # https://github.com/bazelbuild/bazel/blob/0.4.5/src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt#L227
     classpath = ":".join(["${RUNPATH}%s" % (j.short_path) for j in rjars])
     jvm_flags = " ".join([ctx.expand_location(f, ctx.attr.data) for f in jvm_flags])
-    javabin = "%s/%s" % (runfiles_root, ctx.executable._java.short_path)
+    # TODO: Replace the following if/else with just .java_executable_runfiles_path
```

Comment:
do we know when this will be available? is there a relevant issue or PR we can link to it?

Reply:
Would the bazel commit be acceptable?  The property is already in, but I felt it a bit premature to depend on it without it being in a bazel release...and even then, I suspect waiting for a while until folks are generally using those releases would be good:

https://github.com/bazelbuild/bazel/commit/f2075d27ca124156fcd7c01242c552175c0cf145

Reply:
sounds good

Reply:
I added this to the comment, thanks!